kubectl-rook-ceph.sh

```diff
@@ -273,6 +274,91 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+function cluster_health() {
+  end_of_command_parsing "$@" # end of command tree
+
+  check_mon_pods_nodes
+  echo
+  check_mon_quorum
+  echo
+  check_osd_pod_count_and_nodes
+  echo
+  check_all_pods_status
+  echo
+  check_pg_are_active_clean
+  echo
+  check_mgr_pods_status_and_count
+}
+
+function check_mon_pods_nodes() {
+  echo -e "${INFO}Checking if atleast three mon pods are running on different nodes"
```

Comment:
"at least" should be two words throughout
```suggestion
  echo -e "${INFO}Checking if at least three mon pods are running on different nodes"
```